hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4534,3 +4544,10 @@ func reportHostedClusterDeletionDuration(hcluster *hyperv1.HostedCluster, funcCl
 	deletionDuration := funcClock.Since(hcluster.DeletionTimestamp.Time).Seconds()
 	hcmetrics.HostedClusterDeletionDuration.WithLabelValues(hcluster.Namespace, hcluster.Name).Set(deletionDuration)
 }
+
+func hostedClusterReleaseImage(hcluster *hyperv1.HostedCluster) string {
```

Comment:
should this be named controlPlaneReleaseImage to be consistent with API and other [refs in the PR](https://github.com/openshift/hypershift/pull/2848/files#diff-c7d518c1d1c9535b41ec975fb986ce5a031f41a0453780c20659f9e3a5009cf0R17)?

Reply:
As with all things programming, naming is hard.  The _real_ name should be `hostedClusterControlPlaneReleaseImage()`, and the HCP version would be `HostedControlPlaneControlPlaneReleaseImage()`, which is pretty crazy.  Maybe... `hcControlPlaneReleaseImage` and `HCPControlPlaneReleaseImage`?